src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1854,8 +1854,8 @@ public class C
                   <Document><![CDATA[
 public class C
 {
-    public C(int Alice, int Bob) { }
-    public C(string ignored) { }
+    public C(object Alice, object Bob) { }
+    public C(object ignored) { }
```

Comment:
so this case is def interesting.  why did it break?  can you clarify?  i have a guess, but i don't want to go down a wrong path of assumptions.

Reply:
Enum types always ignored the option under test here. The test needed to be updated because `int` (`int.MaxValue`) and `string` (`string.Empty`) both act like enums for triggering here.

Reply:
so my pref would be to take that test and doc the new behavior, and then add a new test for the old scenario personally.  not a huge deal though.

Reply:
but that does seem like an acceptable chance here.

Reply:
➡️ This change was reverted.